92 branch

ukenhk-kuni077 opened the pull request:
ADD category , location listbox to task detail.

Commit:
install jquery

Commit:
jquery test

Commit:
ADD task detail.
But I can't show <select> for location , category.

Commit:
MOD re-format spaces and empty lines.
Git didn't recognize changes , then I changed again.

Commit:
Merge branch '92Branch' of https://github.com/ukenhk-kuni077/blue-vue into 92Branch

ADD category , location listbox to task detail.

Commit:
ADD category , location listbox to task detail.

amanoese commented on src/Task.vue:
```diff
@@ -9,6 +9,19 @@
+        category : <select v-model.sync="task.category">
+          <option>shopping</option>
+          <option>visit</option>
+          <option>other</option>
```
need attribute "value"

amanoese replied:
need attribute "value"

amanoese commented on src/Task.vue:
```diff
@@ -9,6 +9,19 @@
       <p>{{ task.body }}</p>
+      <p>
+        date : <input type="date" v-model.sync="task.date"></input>
+        location : <select v-model.sync="task.location">
```
select tag need label tag.

matteriarize is restrict any html code .
[check for this page](http://materializecss.com/forms.html#select)

Commit:
ADD labels , values for listbox.
I got error message "webkit speech recognition is not defind" with my firefox.
Then , I cannot test this patch.

ukenhk-kuni077 commented:
I pushed new my code.
Please check it out.
Thanks.
